src/Gui/MainWindow.cpp

```diff
@@ -692,18 +692,18 @@ void MainWindow::whatsThis()
 
 void MainWindow::showDocumentation(const QString& help)
 {
-    // temporary - allows to enable/disable the use of the Help module
-    ParameterGrp::handle hGrp = App::GetApplication().GetParameterGroupByPath("User parameter:BaseApp/Preferences/Mod/Help");
-    bool useHelpModule = hGrp->GetBool("UseHelpModule", false);
-    if (useHelpModule) {
+    if (Base::Interpreter().loadModule("Help",true)) {
         Gui::Command::addModule(Gui::Command::Gui,"Help");
         Gui::Command::doCommand(Gui::Command::Gui,"Help.show(\"%s\")", help.toStdString().c_str());
     } else {
         QUrl url(help);
         if (url.scheme().isEmpty()) {
-            QString page;
-            page = QString::fromUtf8("%1.html").arg(help);
-            d->assistant->showDocumentation(page);
+            //QString page;
+            //page = QString::fromUtf8("%1.html").arg(help);
+            //d->assistant->showDocumentation(page);
+            QMessageBox::critical(getMainWindow(), tr("Help addon needed!"),
+            tr("The Help system of FreeCAD is now handled by the \"Help\" addon."
+                "Install it with menu Tools > Addons Manager"));
```

Comment:
What do you think about a button that opens the Addon Manager, possibly even with a filter on "Help"?

Reply:
Excellent idea! How would I start the addons manager that way?

Reply:
Good question... I thought it was possible to get a reference to the actual `CommandAddonManager` class using `Command.get("Std_AddonMgr")`, but it doesn't work the way I thought it did. There are some cheesy ways of doing it (by directly setting some of Addon Manager's preferences, which it uses to store the last filter information), but that's pretty fragile, and has your addon digging into the internals of the Addon Manager, which is obviously not a good idea. Do you know how/if it's possible to get the actual python object that is the instance of class `CommandAddonManager`?

Reply:
All this has no python implementation (yet) i think... probaply better to hack something for now.. but it's not super urgent, it works already, just less smooth